OpenRA.Mods.Common/Traits/Parachutable.cs

```diff
@@ -71,7 +71,8 @@ public void OnLanded()
 			if (sequence != null && palette != null)
 				self.World.AddFrameEndTask(w => w.Add(new Explosion(w, self.OccupiesSpace.CenterPosition, sequence, palette)));
 
-			self.Kill(self);
+			if(self.World.ActorMap.GetActorsAt(self.Location).Equals(ignore))
```

Comment:
`GetActorsAt` returns a collection, so comparing it with `ignore` (an actor) will always be false.  You want something like `self.World.ActorMap.GetActorsAt(self.Location).Any(a => a != ignore)`.  Also space between `if (` please.